tests/integ/test_udtf.py

```diff
@@ -34,6 +38,36 @@
 pytestmark = pytest.mark.udf
 
 
+@pytest.fixture(scope="module")
+def vectorized_udtf_test_table(session) -> str:
```

Comment:
You might need to enable the vectorized udtf in the stored proc test file by adding an `alter session`.

Reply:
The parameter is internal though. Should we set it using `alter session` or on the account level?

Reply:
No. I mean you could add the `alter session` in [this test file](https://sourcegraph.c2.us-central1.gcp-dev.app.snowflake.com/github.com/snowflakedb/snowflake/-/blob/RegressionTests/regressions/t_python_sp_integ/src/snowpark_test_udtf.sql).